pkgs/by-name/xw/xwax/package.nix

```diff
@@ -32,6 +32,11 @@ stdenv.mkDerivation (finalAttrs: {
     # therefore we patch interface.c to also look up in the dejavu_fonts nix store path
     substituteInPlace interface.c \
       --replace-fail "/usr/X11R6/lib/X11/fonts/TTF" "${dejavu_fonts.outPath}/share/fonts/truetype/"
+
+    # make paths to executed binaries hermetic:
+    substituteInPlace import \
+      --replace-fail "exec cdparanoia" "exec ${lib.getBin cdparanoia}/bin/cdparanoia" \
+      --replace-fail "exec ffmpeg" "exec ${lib.getBin ffmpeg}/bin/ffmpeg"
```

Comment:
Shouldn't be using `lib.getExe pkg` instead of `${lib.getBin pkg}/bin/...`?

Reply:
Sounds good. Changed to:

```diff
--- a/pkgs/by-name/xw/xwax/package.nix
+++ b/pkgs/by-name/xw/xwax/package.nix
@@ -35,8 +35,8 @@ stdenv.mkDerivation (finalAttrs: {

     # make paths to executed binaries hermetic:
     substituteInPlace import \
-      --replace-fail "exec cdparanoia" "exec ${lib.getBin cdparanoia}/bin/cdparanoia" \
-      --replace-fail "exec ffmpeg" "exec ${lib.getBin ffmpeg}/bin/ffmpeg"
+      --replace-fail "exec cdparanoia" "exec ${lib.getExe cdparanoia}" \
+      --replace-fail "exec ffmpeg" "exec ${lib.getExe ffmpeg}"
   '';

   buildInputs = [
```